worker/run.py

```diff
@@ -191,6 +192,11 @@ def update_status_and_check_killed(status):
                     self._container_id = do_start()
                 else:
                     raise
+
+            self._image_manager.touch_image(self._resources['docker_image'])
```

Comment:
Just a clarification:
you can't remove images by digest on the REST api, but it is possible on the cli